scripts/gen_berts.py

```diff
@@ -272,8 +272,12 @@ def forward(self, input_ids, token_type_ids=None, attention_mask=None, transform
 		input_ids = input_ids.to(device)
 		attention_mask = attention_mask.to(device)
 		transforms = transforms.to(device)
-		sequence_outputs, pooled_outputs = self.bert.forward(input_ids, token_type_ids=None, attention_mask=attention_mask)
```

Comment:
This casts the output to a `tuple` !

Reply:
Thank you for the contribution (sorry for the delay in reviewing)—reinstalled from scratch today and this works with the example script. Again thank you, PJB